package.json

```diff
@@ -72,6 +72,14 @@
                 "command": "clojureVSCode.evalAndShowResult",
                 "title": "Clojure: Eval and show the result"
             },
+            {
+                "command": "clojureVSCode.testNamespace",
+                "title": "Clojure: Test the current namespace."
+            },
+            {
+                "command": "clojureVSCode.runAllTests",
+                "title": "Clojure: Run All Tests."
```

Comment:
@avli what do you think of the name of these commands?

Reply:
@marcomorain Looks good to me. Personally, I'd prefer "Clojure: Run All Unit Tests", but it's a subjectivity and up to you.

Reply:
Oh, and we shouldn't use dots at the end of commands.